tooling/src/hypothesistooling/projects/hypothesispython.py

```diff
@@ -65,19 +65,21 @@ def has_source_changes():
     return tools.has_changes([PYTHON_SRC])
 
 
-def build_docs(builder="html"):
+def build_docs(*, builder="html", only=[]):
     # See https://www.sphinx-doc.org/en/stable/man/sphinx-build.html
     # (unfortunately most options only have the short flag version)
     tools.scripts.pip_tool(
-        "sphinx-build",
-        "-W",
-        "--keep-going",
-        "-T",
-        "-E",
-        "-b",
-        builder,
-        "docs",
-        "docs/_build/" + builder,
+        *[
+            "sphinx-build",
+            "-W",
+            "-T",
+            "-E",
+            "-b",
+            builder,
+            "docs",
+            "docs/_build/" + builder,
+            *only,
+        ],
```

Comment:
I don't think this needs to be changed?  Creating-and-splatting a list is identical to just listing them as posargs and splatting `*only` directly into the function call.

Reply:
oops, yup!